### DISCUSSION TITLE
How can I achieve the following carousel?

### DISCUSSION THREAD

**Discussion by darshanhtailor** [vadymshymko/react-simply-carousel#224] [Q&A] [answered]
![image](https://github.com/vadymshymko/react-simply-carousel/assets/81902149/f18454cc-f428-4c6a-b6c8-59ea87859927)
The active slide shouldn't be at the center nor at the extreme left. I tried adding a translate in x style in itemsListProps but it isn't working.

**Comment by vadymshymko** [comment, marked as answer]
Hello. You can try something like this:

1. Add some className  to the active slide (first from left side) via
activeSlideProps.className prop
2. Add some className to all slides
3. Add something like this:
.yourClassNameForActiveSlide + .yourClassNameForAllSlide {...some styles

**Reply by darshanhtailor** [reply]
I was able to implement it by adding translate style in itemListProps itself. Maybe I was doing something wrong elsewhere previously.
